lib/core-generators/new/templates/views/pages/legal/privacy.ejs

```diff
@@ -38,17 +38,36 @@
     <p>If you opt-in to a feature that allows third parties to contact you, personal information about you (such as your contact information and other information collected during your visit to the Service) may be shared with data aggregators, marketers, and other organizations (possibly in the form of list rental).</p>
     <p>In the event that we sell, assign or transfer some or all of our business to a successor or acquirer, we may sell, assign or transfer all of your information, regardless of your status, to such successor or acquirer.</p>
 
+    <h3>How long we store information</h3>
+    <p>NEW_APP_COMPANY_NAME will store your data as long as needed to provide you with our services and to operate our organization. If you ask NEW_APP_COMPANY_NAME to delete specific personal information, we'll honor your request unless deleting that information prevents us from carrying out necessary functions, like calculating taxes or conducting required audits.</p>
+
     <h3>Security</h3>
-    <p>We have security measures in place to help protect against the unauthorized access of personally identifiable information under our control. We utilize both online and offline security methods, including firewalls, passwords and restricted physical access to the places where your information is stored to help protect personally identifiable information. Our staff is trained to comply with our security procedures, and our security procedures are regularly reviewed and revised as we deem necessary.</p>
+    <p>We have security measures in place to help protect against the unauthorized access to personally identifiable information under our control. We utilize both online and offline security methods, including firewalls, passwords and restricted physical access to the places where your information is stored to help protect personally identifiable information. Our staff is trained to comply with our security procedures, and our security procedures are regularly reviewed and revised as we deem necessary.</p>
```

Comment:
Let's change "staff" to "team" -- we don't know the details of the organizational structure at play